cdh53/build.xml

```diff
@@ -0,0 +1,16 @@
+<!--===========================================================================
+  This is the build file for the CDH4 Shim modules.
+
+  See ../build-res/subfloor.xml for more details
+============================================================================-->
+<project name="pentaho-hadoop-shims-cdh52" basedir="." default="dist"
```

Comment:
The project should be pentaho-hadoop-shims-cdh52, also you might as well update line 2 to say CDH 5.3 or something...